model/registry/gen-ai.yaml

```diff
@@ -0,0 +1,72 @@
+groups:
+  - id: registry.llm
+    prefix: gen_ai
+    type: attribute_group
+    brief: >
+      This document defines the attributes used to describe telemetry in the context of LLM (Large Language Models) requests and responses.
+    attributes:
+      - id: system
+        type:
+          allow_custom_values: true
+          members:
+            - id: openai
+              value: "OpenAI"
```

Comment:
I believe it should be lowercase
```suggestion
              value: "openai"
```

for two reasons:
- we ask system value to match one used in the namespace (`get_ai.openai.something.special`) and attribute names are lower case
- consistency - all enum values are in lowercase

Unfortunately it's not enforced by tooling and lint checks today, but hopefully will be at some point.